### ISSUE TITLE
Wallet - SSN field can hold only four digits but there is an error asking to enter 9 digit

### ISSUE THREAD

**Issue by izarutskaya** [Expensify/App#46407]
If you haven’t already, check out our [contributing guidelines](https://github.com/Expensify/ReactNativeChat/blob/main/contributingGuides/CONTRIBUTING.md) for onboarding and email contributors@expensify.com to request to join our Slack channel!
___

**Version Number:** v9.0.13-0
**Reproducible in staging?:** Y
**Reproducible in production?:** Y
**Email or phone of affected tester (no customers):** nazrihabtish@gmail.com
**Logs:** https://stackoverflow.com/c/expensify/questions/4856
**Issue reported by:** Applause-Internal team

## Action Performed:
1. Go to account setting > wallet
2. Click on enable wallet
3. verify Personal Info tab opens
4. fill the form with valid form and press next
at the end click on confirm
5. verify it throws error message regarding SSN (this message is what I told you)

## Expected Result:
confirm and proceed to opening the bank account flow

## Actual Result:
error message requiring to input the full 9 digits of SSN while the SSN field is only able to hold 4 digits

## Workaround:
Unknown

## Platforms:

Which of our officially supported platforms is this issue occurring on?
- [ ] Android: Native
- [ ] Android: mWeb Chrome
- [ ] iOS: Native
- [ ] iOS: mWeb Safari
- [x] MacOS: Chrome / Safari
- [ ] MacOS: Desktop

## Screenshots/Videos

![Bug6554647_1722082130142!photo_2024-07-27_15-03-01](https://github.com/user-attachments/assets/5d088bbc-1bc1-49ba-8cbf-694a01ab45ca)

</details>

[View all open jobs on GitHub](https://github.com/Expensify/App/issues?q=is%3Aopen+is%3Aissue+label%3A%22Help+Wanted%22)

**Comment by dangrous** [comment]
Yep, works for me! We should make sure we catch all the places things need to be adjusted but the solution proposal is great.

**Comment by slafortune** [comment]
This was originally @CortneyOfstad  --It's being worked on, it will have a 7-day regression period, she'll be back, so unassigning myself.

**Comment by CortneyOfstad** [comment]
PR is still being worked on so no update 👍

**Comment by dangrous** [comment]
This is on prod, but no automation? Hopefully we get it soon. If not, payment on ... 9/3 I think?

**Comment by CortneyOfstad** [comment]
@dangrous @ishpaul777 sorry for the delay here — to confirm, payment is still okay? If so, I will get that sorted ASAP today (was OoO yesterday). Thanks!

**Comment by dangrous** [comment]
yep as far as I know, there were no regressions, so we're ready for payment

**Comment by dangrous** [comment]
oh but @ishpaul777 can you work through the checklist, even though it didn't post here? Thanks!

**Comment by CortneyOfstad** [comment]
@daledah — sent you an offer via Upwork [here](https://www.upwork.com/nx/wm/offer/103835162)! For some reason, it did not automatically add you (sorry about that)! Please let me know once you accept and I will get that paid ASAP!

@ishpaul777 — you have been paid! Any questions, just let me know!

**Comment by CortneyOfstad** [comment]
## Regression Test 

https://github.com/Expensify/Expensify/issues/426285

**Comment by CortneyOfstad** [comment]
@daledah just bumping you again — thank you!

**Comment by CortneyOfstad** [comment]
## Payment Summary

@ishpaul777 — paid $250 via Upwork
@daledah — paid $250 via Upwork
